development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -197,15 +197,15 @@ following parameters:
 
 ::
 
-    scons platform=linuxbsd tools=no target=release bits=32
-    scons platform=linuxbsd tools=no target=release_debug bits=32
+    scons platform=linuxbsd target=template_release bits=32
```

Comment:
`bits` no longer exist, `arch` is used instead.

Reply:
on linux the bits field is replaced with `x86_64`, is that the same on other platforms (like windows)?

```suggestion
``bin/godot.<platform>.editor.<arch>.mono``, e.g. ``bin/godot.linuxbsd.editor.x86_64.mono``
```

Reply:
i've updated every `bits=32` or `bits=64` occurrence I could find